File: Exercise/Medium/14. Sort Strings Lexicographically/Code/Sort Strings Lexicographically.cs
using System;
using System.Linq;

class Program {
    static void Main() {
        string input = Console.ReadLine();
        
        string[] words = input.Split(' ');
        Array.Sort(words);
        
        string result = string.Join(", ", words);
        Console.WriteLine(result);
    }
}

File: Exercise/Medium/14. Sort Strings Lexicographically/Code/Sort Strings Lexicographically.java
import java.util.Arrays;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String input = scanner.nextLine();

        String[] words = input.split(" ");
        Arrays.sort(words);

        String result = String.join(", ", words);
        System.out.println(result);
    }
}



